test/fail_compilation/mixin.d

```diff
@@ -0,0 +1,26 @@
+// REQUIRED_ARGS: -mixin=test.mixin
+//
+/*
+TEST_OUTPUT:
+---
+test.mixin(7): Error: undefined identifier `b`
```

Comment:
I think the original location of the mixin needs to be mentioned in the error, too. Otherwise it's too inconvenient to jump to the source code to fix the error, especially in an IDE where you expect to double click on the error message only.

Reply:
if you jump to the source it will be 'headered' by `expansion from ...` I'm not sure how to inject an error supplemental, given the error call could come from anywhere.

Reply:
It probably needs something like a mixin location stack, similar to template instantiations. Maybe this can be reused.

Reply:
Maybe, I'd have to look into it, I'm currently rather ... constrained ... at the moment, will be at least a week.